Source/ExecutionEngine/ExecutionEngine.cs

```diff
@@ -175,6 +179,7 @@ protected static string CleanUp(string msg)
 
   public class ExecutionEngine : IDisposable
   {
+    // Should it be set per request?
```

Comment:
It's moved into ExecutionEngineOptions in this PR: https://github.com/boogie-org/boogie/pull/520

Could you remove the comment?